samples/hello++/main.cpp

```diff
@@ -1,27 +1,21 @@
 #include <hal/debug.h>
 #include <hal/video.h>
+#include <string>
+#include <vector>
 #include <windows.h>
 
-class NumberClass{
-private:
-  int hiddenValue = 0;
-public:
-  NumberClass() {hiddenValue = 5;};
-  NumberClass(int v) : hiddenValue(v) {};
-  int getValue() { return hiddenValue; };
-};
-
 int main(void) {
   XVideoSetMode(640, 480, 32, REFRESH_DEFAULT);
+  std::vector<std::string> words;
+  words.emplace_back("Hello");
+  words.emplace_back(" ");
+  words.emplace_back("World\n");
 
-  NumberClass defaultFive;
-  NumberClass customSix(6);
-
-  while (true) {
-    debugPrint("Value of defaultFive: %i\n", defaultFive.getValue());
-    debugPrint("Value of customSix  : %i\n", customSix.getValue());
-    Sleep(2000);
+  while (1) {
+    for (auto& word : words) {
+      debugPrint("%s", word.c_str());
+      Sleep(2000);
```

Comment:
I'd probably move this out of the loop? Especially because " " would not be a visible addition